dapps/marketplace/src/components/growth/Action.js

```diff
@@ -372,7 +363,7 @@ require('react-styl')(`
       .image-holder
         position: relative
       .title
-        font-size: 18px
+        font-size: 1.31rem
```

Comment:
I have been reading this article: https://engageinteractive.co.uk/blog/em-vs-rem-vs-px suggesting that setting font sizes in `rem` is more user friendly, since it respects the default font size user has set for the browser. Also scaling up the whole project fonts is supposedly easier since `rems` are tied to font size of html root element. 

You think they should be set to `px` in favour of consistency?

Reply:
I think px for font sizes for now, until we replace them with css vars, then padding and spacing with rems

Reply:
Cool changed to px across the pull request

Reply:
Hmm I don't see this being updated yet

Reply:
Ah missed this one... corrected now